packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1834,10 +1834,11 @@ In case `enum` is used in conjunction with `oneOf`, the set of enumerated values
 
 Okta has also extended [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04) with the following keywords:
 
-| Property      | Description                                     | DataType                                                                  | Nullable | Unique | Readonly |
-| :------------- | :----------------------------------------------- | :------------------------------------------------------------------------- | :--------- | :------ | :-------- |
-| required      | determines whether the property is required     | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
-| permissions   | access control permissions for the property     | Array of [Schema property permission](#schema-property-permission-object) | FALSE    | FALSE  | FALSE    |
+| Property                                                        | Description                                                       | DataType                                                                  | Nullable | Unique | Readonly |
+| :-------------------------------------------------------------- | :--------------------------------------------------------------- | :------------------------------------------------------------------------ | :------- | :----- | :------- |
+| required                                                        | Determines whether the property is required                       | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
+| permissions                                                     | Access control permissions for the property                       | Array of [Schema property permission](#schema-property-permission-object) | FALSE    | FALSE  | FALSE    |
+| [unique](#unique-attribute) <ApiLifecycle access="ea" />        | Determines whether the property enforces uniqueness constraint    | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
```

Comment:
For ease of review:
![Screen Shot 2021-06-30 at 12 24 23 PM](https://user-images.githubusercontent.com/12566503/124019861-6fe0dd80-d99e-11eb-87c0-0f38dbb678d8.png)